config/tracks/acu_bathurst.ini

```diff
@@ -1,251 +1,257 @@
+CONDITION = NIGHT_SMOOTH
+KEY_0 = ksEmissive
+VALUE_0 = 1,1,1, 25
+VALUE_0_OFF = 0, 0, 0
+ACTIVE = 1
+
+[MATERIAL_ADJUSTMENT_1]
+DESCRIPTION = sponsor bridges and billboards
+MESHES = obj-sponsor-??
+CONDITION = NIGHT_SMOOTH
+KEY_0 = ksEmissive
+VALUE_0 = 70, 50, 30, 0.015
+VALUE_0_OFF = 0, 0, 0
+ACTIVE = 1
+VISIBILITY_LEVEL = 1
+
+[MATERIAL_ADJUSTMENT_7]
+DESCRIPTION = road billboards
+MATERIALS = BAT_CWALL_02, BAT_CWALL_20
+CONDITION = NIGHT_SMOOTH
+KEY_0 = ksEmissive
+VALUE_0 = 70, 50, 30, 0.015
+VALUE_0_OFF = 0, 0, 0
+ACTIVE = 1
+VISIBILITY_LEVEL = 1
+
+[MATERIAL_ADJUSTMENT_8]
+DESCRIPTION = other buildings
+MATERIALS = BLD_BUILDINGC,BLD_MUSEUM,BLD_BUILDING04A,BLD_BUILDING0106,BLD_BUILDING39,BLD_BUILDING0106,BLD_BUILDING44,BLD_HOUSE46,BLD_HOUSE48A,BLD_WINEHOUSEZ,BLD_HOUSE18A,BLD_HOUSE18B,BLD_HOUSE13A,BLD_AH_BBUILDING04_02
+CONDITION = NIGHT_SMOOTH
+KEY_0 = ksEmissive
+VALUE_0 = 1, 1, 1, 0.2
+VALUE_0_OFF = 0, 0, 0
+ACTIVE = 1
+VISIBILITY_LEVEL = 2
+
+[MATERIAL_ADJUSTMENT_2]
+DESCRIPTION = main building windows, other windows
+MATERIALS = PITBUILDINGS_A, BLD_AH_BBUILDING04_07, BLD_BUILDING34C
+CONDITION = NIGHT_SMOOTH
+KEY_0 = ksEmissive
+VALUE_0 = 10, 181, 255, 0.005
+VALUE_0_OFF = 0, 0, 0
+ACTIVE = 1
+VISIBILITY_LEVEL = 1
+
+[MATERIAL_ADJUSTMENT_6]
+DESCRIPTION = main building writes
+MATERIALS = building_pit_a_06, NUMBERS1
+CONDITION = NIGHT_SMOOTH
+KEY_0 = ksEmissive
+VALUE_0 = 1, 1, 1, 3
+VALUE_0_OFF = 0, 0, 0
+ACTIVE = 1
+VISIBILITY_LEVEL = 0
+
+[MATERIAL_ADJUSTMENT_3]
+DESCRIPTION = videowalls
+MATERIALS = VIDEOWALLB_NH
+CONDITION = RBGYW
+KEY_0 = ksEmissive
+VALUE_0 = 125, 252, 0, 0.03
+VALUE_0_OFF = 0, 0, 0
+ACTIVE = 1
+VISIBILITY_LEVEL = 1
+
+[MATERIAL_ADJUSTMENT_9]
+DESCRIPTION = videowalls mount panorama
+MATERIALS = MP_VIDEOWALL
+CONDITION = RBGYW
+KEY_0 = ksEmissive
+VALUE_0 = 125, 252, 0, 0.05
+VALUE_0_OFF = 0, 0, 0
+ACTIVE = 1
+VISIBILITY_LEVEL = 1
+
+[MATERIAL_ADJUSTMENT_4]
+DESCRIPTION = 'mount panorama' on the hill, bridges, signal stations
+MATERIALS = bat_mt_panorama_01,PITBUILDINGS_B,PITBUILDINGS_D,BLD_SIGNALSTATION04
+CONDITION = NIGHT_SMOOTH
+KEY_0 = ksEmissive
+VALUE_0 = 1, 1, 1, 4
+VALUE_0_OFF = 0, 0, 0
+ACTIVE = 1
+
+[MATERIAL_ADJUSTMENT_10]
+
+MATERIALS=GRASS38?
+KEY_0=ksDiffuse
+VALUE_0=0.2
+OFF_VALUE_0=0.25
+KEY_1=ksAmbient
+VALUE_1=0.40
+OFF_VALUE_1=0.25
+
+[MATERIAL_ADJUSTMENT_11]
+
+MATERIALS=3Dgrass?
+KEY_0=ksDiffuse
+VALUE_0=0.2
+OFF_VALUE_0=0.25
+KEY_1=ksAmbient
+VALUE_1=0.10
+OFF_VALUE_1=0.25
+
+[MATERIAL_ADJUSTMENT_12]
+
+MATERIALS=gumtree02,gumtree03,treeline,WINEPLANTS_LO1,trees_a,bush01?
+KEY_0=ksDiffuse
+VALUE_0=0.175
+OFF_VALUE_0=0.25
+KEY_1=ksAmbient
+VALUE_1=0.25
+OFF_VALUE_1=0.25
+
+[MATERIAL_ADJUSTMENT_13]
+
+MATERIALS=terrain1?
+KEY_0=ksDiffuse
+VALUE_0=0.2
+OFF_VALUE_0=0.25
+KEY_1=ksAmbient
+VALUE_1=0.22
+OFF_VALUE_1=0.25
+
+[MATERIAL_ADJUSTMENT_14]
+
+MATERIALS=gumtree01?
+KEY_0=ksDiffuse
+VALUE_0=0.20
+OFF_VALUE_0=0.25
+KEY_1=ksAmbient
+VALUE_1=0.09
+OFF_VALUE_1=0.25
```

Comment:
Note: This is an old config so this may or may not be your fault

Shader replacements are the best practice for fixed adjustments, not a terribly big inconvenience but it would be preferred if you could readjust the code here or bind it to a seaonal condition as other "de-neonizing" configs do.

Reply:
Indeed i did not touch this part of the config, i'll have a look at the other configs though to see if i can fix it!

Reply:
has been addressed